include/miral/miral/input_configuration.h

```diff
@@ -139,6 +148,10 @@ class InputConfiguration::Keyboard
     void set_repeat_rate(int new_rate);
     void set_repeat_delay(int new_delay);
 
+    /// Copies the corresponding value from `other` for all unset fields.
+    /// \remark Since MirAL 5.2
```

Comment:
```suggestion
    /// \remark Since MirAL 5.3
```

Reply:
Actually, this is redundant: the whole of InputConfiguration::Keyboard is already "Since MirAL 5.3"